k9mail/src/main/java/security/planck/ui/passphrase/manage/compose/PassphraseManagementScreen.kt

```diff
@@ -158,8 +140,8 @@ fun RenderManagingAccounts(
         defaultColor = defaultColor,
         errorColor = errorColor,
         successColor = successColor,
-        validateInput = validateNewPassphrase,
-        verifyNewPassphrase = verifyNewPassphrase,
+        validateInput = { validateInput(state.oldPasswordStates.size, it) },
+        verifyNewPassphrase = { validateInput(state.oldPasswordStates.size + 1, it) },
```

Comment:
is the +1 the positional index of the value in the array of values?

Reply:
yes, all the values are oldPasswordStates + newPasswordState + newPasswordVerificationState.

Reply:
great